lib/plug/templates/debugger.html.eex

```diff
@@ -707,14 +707,13 @@
         <% end %>
 
         <header class="exception-info">
-            <% [headline | details] = String.split(@message, "\n\n") %>
             <h5 class="struct">
                 <%= h @title %>
                 <small>at <%= h method(@conn) %></small>
                 <small class="path"><%= h @conn.request_path %></small>
             </h5>
-            <h1 class="title"><%= h headline %></h1>
-            <%= for detail <- details do %>
+            <code class="headline"><pre><%= h @headline %></pre></code>
```

Comment:
Using `<pre>` to represent multi lines as noted here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code#notes